onnx/onnxifi_dummy.c

```diff
@@ -136,3 +141,71 @@ ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
 onnxReleaseGraph(onnxGraph graph) {
   return ONNXIFI_STATUS_SUCCESS;
 }
+
+/*
+ * ONNXIFI Extension Functions
+ */
+
+ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
+onnxGetExtensionFunctionAddress(
+    onnxBackendID backendID,
+    const char* name,
+    onnxExtensionFunctionPointer* function) {
+  /*		const int extension_function_number = 2;
```

Comment:
Where is `extension_function_number` defined? It is commented out here?

Reply:
oh it's my bad, I thought I have already deleted this.